src/Form.js

```diff
@@ -0,0 +1,186 @@
+import React, { useState, Fragment } from 'react';
+import CustomDate from './CustomDate';
+import GenderOptions from './GenderOptions';
+import LevelOfEducation from './LevelOfEducation';
+import mockData from './data.json';
+import { nanoid } from 'nanoid';
+import ReadOnlyRow from './ReadOnlyRow';
+import EditRow from './EditRow'
+
+function Form() {
+    const [inputData, setInputData] = useState({
```

Comment:
const initialValue = {
        name: "",
        email: "",
        birthdate: "",
        gender: "",
        education: "",
        password: ""
    }

use this variable